packages/eslint/README.md

```diff
@@ -0,0 +1,68 @@
+# eslint-plugin-webassembly
+
+ESLint plugin for WebAssembly
+
+## Installation
+
+You'll first need to install [ESLint](http://eslint.org):
+
+```
+$ npm i eslint --save-dev
+```
+
+Next, install `eslint-plugin-webassembly`:
+
+```
+$ npm install eslint-plugin-webassembly --save-dev
+```
+
+**Note:** If you installed ESLint globally (using the `-g` flag) then you must also install `eslint-plugin-webassembly` globally.
+
+## Usage
+
+Add `webassembly` to the plugins section of your `.eslintrc` configuration file. You can omit the `eslint-plugin-` prefix:
+
+```json
+{
+    "plugins": [
+        "webassembly"
+    ]
+}
+```
+
+
+Then configure the rules you want to use under the rules section.
+
+```json
+{
+    "rules": {
+        "webassembly/non-unexisting-export": 2
+    }
+}
+```
+
+## Supported Rules
+
+### `no-unexisting-export`
```

Comment:
I must admit, I'm not keen on that naming. How about `no-missing-exports`? or `no-exports-missing`?

Reply:
I agree, what about `no-unknown-export`?

Reply:
yeah, `no-unknown-export` sounds good too